src/rtc/geogrid.py

```diff
@@ -406,15 +406,10 @@ def generate_geogrids_from_db(bursts, geo_dict, mosaic_dict, burst_db_file):
         epsg_mosaic, product_mosaic_str)
 
 
-    for burst_id, burst_pol in bursts.items():
-        pol_list = list(burst_pol.keys())
-        burst = burst_pol[pol_list[0]]
+    for burst_id, _ in bursts.items():
         if burst_id in geogrids_dict.keys():
             continue
 
-        radar_grid = burst.as_isce3_radargrid()
-        orbit = burst.orbit
-
```

Comment:
There was a small chain reaction about removing the variables. What I've initially found was that `radar_grid` and `orbit` were defined but not accessed in the for loop.
Both of the area from the variable `burst` which is from the code below in the loop:

```python
    for burst_id, burst_pol in bursts.items():
        pol_list = list(burst_pol.keys())
        burst = burst_pol[pol_list[0]]
```
where 
`burst_pol` is from the for statement, and it was only used to define `burst`

Therefore:
- removed the unused variable `radar_grid` and `orbit`
- The removal above made `pol_list` and `pol_burst` unused, therefore removed them as well
- `pol_list` and `pol_burst` originates from `burst_pol` which is extracted from the for statement. I do not see any uses of the variable `burst_pol` after removing the variables above, so I replaced that with `_` in the for statement.

The test went through after the modification. And I do see that the PR has passed the CI test.

Reply:
Hi @seongsujeong , sounds good. I saw `radar_grid` and `orbit` were used around lines 622, but I just noticed that those variables referred to other `radar_grid` and `orbit` variables (defined around line 614, rather than 415). It should be fine then. Thanks for the update.